src/program/services/scrapers/peerflix.py

```
@@ -0,0 +1,124 @@
+        except Exception as e:
+            logger.error(
+                f"Peerflix failed to initialize: {e}",
+            )
+            return False
+        return True
+
+    def run(self, item: MediaItem) -> Dict[str, str]:
+        """Scrape Peerflix with the given media item for streams"""
+        try:
+            return self.scrape(item)
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)